test_pull_requests

```diff
@@ -1142,7 +1341,7 @@ popd
               if JenkinsAPI.is_project_stable?(branch, settings)
                 submit_test_job = true
               else
-                updated_comment = "#{settings['test_prefix']} Waiting for stable build of '#{branch_settings(branch, settings)['downstream_job_name']}'"
+                updated_comment = compose_bot_comment(settings['test_prefix'], :content => waiting_for_stable_build(branch, settings), :state => :wait_stable_build)
```

Comment:
if we have a one-to-one mapping do we still need to pass both the content and the state?